theme/eva.js

```diff
@@ -0,0 +1,80 @@
+const { default: Eva } = require('@eva-design/eva');
+
+module.exports = {
+    ...Eva,
```

Comment:
Sorry, I didn't totally track: what does spreading `Eva` do here? I assumed it wasn't a theme object based on usage elsewhere, but sounds like no? it's an object with keys that follow this color pattern?

Reply:
It turns out this isn't needed here, so I got rid of it! `Eva` is the full theme object, which includes a default color palette but also some `light` and `dark` palettes which we aren't messing with yet. It's also being passed to Eva's `ApplicationProvider` in App.js so we don't need it here.